.pylintrc

```diff
@@ -0,0 +1,2 @@
+[Format]
+max-line-length=80
```

Comment:
It's already not a hard requirement: you are not expected to have 10/10 on pylint. Instead, you only have to meet 8.5/10. I think I got all of the tools I touched above 8.75, leaving some wiggle room.